### ISSUE TITLE
Scroll Further than the last line

### ISSUE THREAD

**Issue by villamilmedia** [zadam/trilium#198]
Hello, First of all let me tell you that Trillium is super awesome ! I'm using Trillium on my MacOS 

The issue/Request that I have is that I'd like to scroll further down a document when typing. 
At the moment when I'm typing something on the bottom line of my screen I can't scroll further due to the scoll bar that ends there.

**Comment by zadam** [comment]
Wow, you're the first user reporting that Trilium works for them in MacOS! (I've never tried it personally)

To your question: I think this is the typical behavior in text editing apps though. Is it because it is annoying to write text on the bottom and you want to move it to e.g. center of the screen? I guess you're aware of an easy workaround of just adding few empty lines after your last line - with this you will be able to scroll down.

**Comment by villamilmedia** [comment]
@zadam yeah it works great on Mac, hopefully Trillium becomes bigger so that support will be as well. I was looking for a good personal Wiki system and when I saw Trillium I directly fall in love. 

For the text issue, it is indeed annoying to write at the bottom of the screen instead of in the center of the screen but I will place some empty lines beneath it.

**Comment by JordyScript** [comment]
> @zadam yeah it works great on Mac, 

Perhaps it would be helpful to other MacOS users if you could outline the steps you took to get Trilium to work here, such that they can also try installing it the way you did.

**Comment by code-ink-space** [comment]
This is something I really wanted to have ever since I started using Trilium over a year ago. I use the Journal heavily where I often end up typing near the bottom of the screen. Manually adding those empty lines has become almost second-nature to me.
But today I cobbled together some workaround with this [JS](https://github.com/code-ink-space/trilium-customizations/blob/main/trilium-editorAutoSpacer.js) and [CSS](https://github.com/code-ink-space/trilium-customizations/blob/main/trilium-editorAutoSpacer.css). My solution auto-scrolls the editor when the cursor goes down to a certain point. The effect is called Typewriter mode in some text editor apps.
One limitation of my solution is it only works with ordinary text notes. I have not tried making it work with other note types and frankly I don't see myself needing this feature there. But having this in the regular text notes is a big deal for me and I hope this little script could be of some help.
<video src='https://user-images.githubusercontent.com/47141674/158040787-290b4f1b-afee-49dc-904d-2931c7887526.mp4' width=400/>
